testcases/testcase_blood_flow_model.py

```diff
@@ -14,53 +14,52 @@
-                                   # 4: todo import graph from edge_data and vertex_data pickle files
-        "write_network_option": 1,  # 1: do not write anything
-                                    # 2: write to igraph format # todo: handle overwriting data from import file
-                                    # 3-...: todo other file formats.
-        "tube_haematocrit_option": 2,  # 1: No RBCs (ht=0)
-                                       # 2: Constant haematocrit
-                                       # 3: todo: RBC tracking
-                                       # 4-...: todo: steady state RBC laws
-        "rbc_impact_option": 3,  # 1: No RBCs (hd=0)
-                                 # 2: Laws by Pries, Neuhaus, Gaehtgens (1992)
-                                 # 3: Laws by Pries and Secomb (2005)
-                                 # 4-...: todo: Other laws. in vivo?
-        "solver_option": 1,  # 1: Direct solver
-                             # 2: PyAMG solver
-                             # 3-...: other solvers
+        "read_network_option": 2,   # 1: generate hexagonal graph
+                                    # 2: import graph from csv files
+                                    # 3: import graph from igraph file (pickle file)
+                                    # 4: todo import graph from edge_data and vertex_data pickle files
+        "write_network_option": 1,      # 1: do not write anything
+                                        # 2: write to igraph format # todo: handle overwriting data from import file
+                                        # 3-...: todo other file formats.
+        "tube_haematocrit_option": 2,   # 1: No RBCs (ht=0)
+                                        # 2: Constant haematocrit
+                                        # 3: todo: RBC tracking
+                                        # 4-...: todo: steady state RBC laws
+        "rbc_impact_option": 3,     # 1: No RBCs (hd=0)
+                                    # 2: Laws by Pries, Neuhaus, Gaehtgens (1992)
+                                    # 3: Laws by Pries and Secomb (2005)
+                                    # 4-...: todo: Other laws. in vivo?
+        "solver_option": 1,     # 1: Direct solver
+                                # 2: PyAMG solver
+                                # 3-...: other solvers
 
         # Blood properties
         "ht_constant": 0.3,  # only required if RBC impact is considered
         "mu_plasma": 0.0012,
 
         # Hexagonal network properties. Only required for "read_network_option" 1
-        "nr_of_hexagon_x": 3,
-        "nr_of_hexagon_y": 3,
+        "nr_of_hexagon_x": 7,
+        "nr_of_hexagon_y": 7,
         "hexa_edge_length": 62.e-6,
         "hexa_diameter": 4.e-6,
         "hexa_boundary_vertices": [0, 27],
-        "hexa_boundary_values": [2, 1],
+        "hexa_boundary_values": [5, 2],
         "hexa_boundary_types": [1, 1],
```

Comment:
Can you please leave the "default" options here?
"nr_of_hexagon_x": 3,
 "nr_of_hexagon_y": 3,
"hexa_boundary_values": [2, 1],